lib/tasks/github_repo_issues_to_author.rake

```diff
@@ -0,0 +1,17 @@
+require 'net/http'
```

Comment:
This is the rake task to simply populate this static repo issues into authors, it has three problems in my opinion:
1- muting an exception: i have done that intentionally not to break the reviewer console.
2- It's calling "opened" directly which will require changes when github decides to change the name into something else, to solve that we can create a hash in the big service with the key that we want and the action of the github, so when they change it we only modify one class "source of truth"
3- static URL for the github repo

Reply:
Got it thanks for the clarifications 👍